dockers/docker-database-global/Dockerfile.j2

```diff
@@ -0,0 +1,10 @@
+FROM docker-database
```

Comment:
Can we even avoid creating the docker-database-global image altogether? Just use docker-database image and when the docker-database-global container is started, it is started with appropriate env variables.

Reply:
current slave.mk doesn't support sharing image for new container, and per-instance container doesn't fit globaldb case. I made the change to support using docker-database image, it appears to work in initial test, but some issues with per-instance database container are still under investigation.